src/test/ui/const-generics/float-generic.adt_const_params.stderr

```diff
@@ -0,0 +1,11 @@
+error[E0741]: `f32` is forbidden as the type of a const generic parameter
+  --> $DIR/float-generic.rs:5:17
+   |
+LL | fn foo<const F: f32>() {}
+   |                 ^^^
+   |
+   = note: equality is not well-defined on floating-point numbers
```

Comment:
I'm not sure if this is more misleading... open to suggestions :smile_cat:

Reply:
maybe we could rely on the fact that the type isn't `Ord`/`Eq` and let people read up on `Ord`/`Eq` themselves?